src/control/ballHandling/DribbleForwards.cpp

```diff
@@ -162,12 +162,6 @@ RobotCommand DribbleForwards::sendDribbleForwardsCommand() {
     interface::Input::drawData(interface::Visual::BALL_HANDLING, {forwardsDribbleLine.first, forwardsDribbleLine.second},
             Qt::white, robot->id, interface::Drawing::LINES_CONNECTED);
 
-    // check if the ball is not too far right or too far left of the robot, and try to compensate for that
-    if (ball->visible && false) {
-        Angle ballAngleRelativeToRobot = (ball->pos - robot->pos).toAngle() - robot->angle;
-        command.vel += (robot->angle + M_PI_2).toVector2(ballAngleRelativeToRobot*0.23);
-    }
-
     // limit velocity close to the target
```

Comment:
Did you talk to Thijs about this change? I'm fine with it because this seems like a hack, but it's definitely a functionality change.

Reply:
Its definitely not a functionality change since if(false) does not do anything ;-)